landoapi/app.py

```diff
@@ -1,19 +1,24 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
+import json
 import os
 
 import click
 import connexion
 import logging
 
 from connexion.resolver import RestyResolver
+from raven.contrib.flask import Sentry
+
 from landoapi.dockerflow import dockerflow
 from landoapi.models.storage import alembic, db
 from mozlogging import MozLogFormatter
 
 logger = logging.getLogger(__name__)
 
+sentry = Sentry()
```

Comment:
It seems like `sentry` can be used in other files to manually handle exceptions, it is a global like `logger`.